verticapy/utilities.py

```diff
@@ -1657,15 +1792,17 @@ def set_option(option: str, value: Union[bool, int, str] = None):
             if res:
                 verticapy.options["temp_schema"] = str(value)
             else:
-                raise ParameterError(
-                    "The schema '{}' could not be found.".format(value)
-                )
+                raise ParameterError(f"The schema '{value}' could not be found.")
     elif option == "time_on":
         check_types([("value", value, [bool])])
-        if isinstance(value, bool):
+        if value in (True, False, None):
```

Comment:
Just wondering, doesn't `check_types([("value", value, [bool])])` implies that `value` is in `(True, False, None)`?

Reply:
It only raises a warning and not an error.